.github/workflows/github-action-scan.yml

```diff
@@ -4,12 +4,20 @@ name: Build SecHub GHA (scan)
 on:
```

Comment:
Please rename your branch according to https://mercedes-benz.github.io/sechub/latest/sechub-techdoc.html#branching-model (I opened an issue to transfer the documentation to the developer guide or contributing.md)
We can also discuss about a better naming convention, but it should be consistent.